### DISCUSSION TITLE
Play File

### DISCUSSION THREAD

**Discussion by axellotl22** [discord-php/DiscordPHP#1067] [Q&A] [answered]
hi there,
could you provide an example for the bot to play a mp3 file on php 8?
i don´t get it to run. i hope someone could help me out here :)

**Comment by SQKo** [comment, marked as answer]
```php
$discord->joinVoiceChannel($channel)->then(function (VoiceClient $voice) { // join voice channel

    $voice->playFile('sample-15s.mp3')->then(null, function ($e) { // sample-15s.mp3 is located relatively to your main bot.php file
        echo 'ERROR: '.$e->getMessage();
    })->done();
})->done();
```

where `$channel` is a voice channel

But you do not need to always `joinVoiceChannel()` if the bot is already in the channel.
